docs/it/token_endpoint.rst

```diff
@@ -233,16 +213,13 @@ L’ID Token è un JSON Web Token (JWT) che contiene informazioni sull’utente
 	    } 
      - 
    * - **exp**
-     - UNIX Timestamp     
-     - Data/ora di scadenza del token in formato NumericDate, come indicato in RFC 7519 – JSON Web Token (JWT), secondo le modalità definite dall’Agenzia per l’Italia Digitale.
+     - UNIX Timestamp. Data/ora di scadenza del token in formato NumericDate, come indicato in RFC 7519 – JSON Web Token (JWT), secondo le modalità definite dall’Agenzia per l’Italia Digitale.
```

Comment:
- UNIX Timestamp con l’istante di scadenza del JWT, codificato come NumericDate come indicato in
       `RFC7519 - JSON Web Token (JWT) <https://www.rfc-editor.org/rfc/rfc7519.html>`_